data/hai/hai reveal 3 early.txt

```diff
@@ -267,12 +279,12 @@ mission "Hai Building Mountaintop [2] Recruits"
 				`	"What's the problem?"`
 					goto problem
 				`	"The movie production is real?"`
-			`	She looks blinks at you as if caught slightly off-guard. "Yes, didn't they tell you? In order to keep up the cover they actually have to start making a movie, or at least believable clips from one. The director they found, an eccentric chap called David Cohen, seems to have got quite enthusiastic now he's not busy bemoaning the 'big reveal' of his 'secret project'. There's a Mr Turner who's now the cover story's guy bankrolling the thing.`
+			`	She looks blinks at you as if caught slightly off-guard. "Yes, didn't they tell you? In order to keep up the cover they actually have to start making a movie, or at least believable clips from one. The director they found, an eccentric chap called David Cohen, seems to have got quite enthusiastic now he's not busy bemoaning the 'big reveal' of his 'secret project.' There's a Mr Turner who's now the cover story's guy bankrolling the thing.`
```

Comment:
I'm not sure whether this part should be changed, what you've done looks correct to me but I've been told to leave them as it is.

Reply:
I'll leave my change in, and see who comments about it.

Reply:
"She looks blinks" doesn't sound correct.

Reply:
> "She looks blinks" doesn't sound correct.

Indeed. Thanks for noticing that. I'll fix it in my next round of textual corrections in 5-10 hours. Please let me know if anything else is wrong.

Reply:
It appears an earlier commit already changed that to "looks and blinks" but github didn't update this comment.

```
`       She looks and blinks at you as if caught slightly off-guard. "Yes, didn't they tell you?
```